include/sentry.h

```diff
@@ -2434,10 +2436,31 @@ SENTRY_API sentry_value_t sentry_value_new_user_feedback_n(
     const char *email, size_t email_len, const char *comments,
     size_t comments_len);
 
+/**
+ * Captures a deprecated User Report and sends it to Sentry.
+ * Deprecated: Please use `sentry_capture_feedback` instead.
+ */
+SENTRY_API void sentry_capture_user_feedback(sentry_value_t user_report);
```

Comment:
`sentry_capture_feedback(sentry_value_t)` vs. `sentry_capture_user_feedback(sentry_value_t)`
   - Both operate on a `sentry_value_t`. Technically, it could be one function that checks for the existence of "message" vs. "comments", for example.
   - I renamed the old argument to `user_report` in some places to help highlight that it operates on the old User Report API.